traveltimepy/mapper.py

```diff
@@ -776,8 +772,8 @@ def create_proto_request(
     properties: Optional[List[PropertyProto]],
     travel_time: int,
     one_to_many: bool = True,
-) -> TimeFilterFastRequest_pb2.TimeFilterFastRequest:
-    request = TimeFilterFastRequest_pb2.TimeFilterFastRequest()
+) -> TimeFilterFastRequest_pb2.TimeFilterFastRequest:  # type: ignore
+    request = TimeFilterFastRequest_pb2.TimeFilterFastRequest()  # type: ignore
 
```

Comment:
`# type: ignore` is used everywhere where proto is called. Without this, `mypy` complains, because the `pb2` file structure is not what it expects